shenyu-client/shenyu-client-tars/src/main/java/org/apache/shenyu/client/tars/TarsServiceBeanEventListener.java

```diff
@@ -62,68 +63,89 @@ public class TarsServiceBeanEventListener implements ApplicationListener<Context
 
     private final String ipAndPort;
 
-    private final String host;
-
-    private final int port;
+    private final Boolean isFull;
 
     public TarsServiceBeanEventListener(final PropertiesConfig clientConfig, final ShenyuClientRegisterRepository shenyuClientRegisterRepository) {
+        super(clientConfig, shenyuClientRegisterRepository);
         Properties props = clientConfig.getProps();
         String contextPath = props.getProperty(ShenyuClientConstants.CONTEXT_PATH);
-        this.host = props.getProperty(ShenyuClientConstants.HOST);
         String port = props.getProperty(ShenyuClientConstants.PORT);
-        if (StringUtils.isAnyBlank(contextPath, this.host, port)) {
+        if (StringUtils.isAnyBlank(contextPath, this.getHost(), port)) {
             throw new ShenyuClientIllegalArgumentException("tars client must config the contextPath, ipAndPort");
         }
         this.contextPath = contextPath;
-        this.ipAndPort = this.host + ":" + port;
-        this.port = Integer.parseInt(port);
+        this.ipAndPort = this.getHost() + ":" + port;
+        this.isFull = Boolean.parseBoolean(props.getProperty(ShenyuClientConstants.IS_FULL, Boolean.FALSE.toString()));
```

Comment:
please remove `ifFull`, `ifFull` is for application level registration. However, because `RPC needs service, method and paramTypes` for `generalized calls`, RPC is not suitable for using the `ifFull` attribute.